### PR TITLE
Fuji's Struct mod to mainline

### PR DESCRIPTION
**Pull request by NotFuji**
#### Summary 

```SUMMARY:  Mods "Added Fuji's Struct mod"```

_10 new houses_

_7 new city buildings:_
- 2 Apartments
- Oudoorsman’s Store
- Car Dealership
- Diner
- Gaming Store
- Gun Store

_4 new specials:_
- Airport
- Light industrial park
- Gas station (with underground bunker)
- Scavenger’s Bunker (with new scenario + 2 professions)

_2 new professions:_
- Rookie Scavenger
- Novice Scavenger

**Requires More_Locations**

### CODE REVIEW

File: data/mods/Fuji_Structures/npc/classes.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+                                { "constant" : 0 }
+                            ]
+                        }
+                    ]
+                }
+            },
+            { "skill": "gun", "bonus" : { "rng" : [ 1, 5 ] } },
+            { "skill": "rifle", "bonus" : { "rng" : [ 1, 5 ] } },
+            { "skill": "pistol", "bonus" : { "rng" : [ 1, 4 ] } },
+			{ "skill": "smg", "bonus" : { "rng" : [ 1, 3 ] } },
```
Comment (cosmiccoincidence) [inline data/mods/Fuji_Structures/npc/classes.json]: Linting your files will save space, make things look nicer and save time for people to move this to vanilla later on.

File: data/mods/Fuji_Structures/worldgen/lightindustry/s_lightindustry_road_1.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+        "aaab          baaaaaaaaa",
+        "bbbb          baaaaaaaaa"
+      ],
+      "terrain": {
+        "a": [ "t_grass", "t_grass", "t_grass", "t_dirt" ],
+        "b": "t_sidewalk",
+        "c": "t_pavement_y",
+        "d": "t_shrub"
+      },
+      "furniture": {}
```
Comment (cosmiccoincidence) [inline data/mods/Fuji_Structures/worldgen/lightindustry/s_lightindustry_road_1.json]: Again this line is not needed.

File: data/mods/Fuji_Structures/worldgen/airport/s_air_atc.json [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+      "furniture": { "h": "f_table", "q": "f_dumpster" },
+	  "gaspumps": { "l": {  } },
+      "place_loot": [
+        { "group": "office_mess", "chance": 80, "repeat": [ 3 ], "x": [ 10, 13 ], "y": [ 2, 5 ] },
+        { "group": "office_mess", "chance": 80, "repeat": [ 1 ], "x": [ 11, 13 ], "y": [ 7, 7 ] },
+        { "group": "trash", "chance": 50, "repeat": [ 5 ], "x": [ 1, 2 ], "y": [ 6, 8 ] }
+      ]
+    }
+  }
+]
```
Comment (cosmiccoincidence) [inline data/mods/Fuji_Structures/worldgen/airport/s_air_atc.json]: Missing blank line at end of this file (which is required). Same goes for most of the other files in this PR.

**Comment by ZhilkinSerg** [comment]
Jenkins rebuild.

**Comment by ZhilkinSerg** [comment]
@cosmiccoincidence I've linted all files in #25667, so it will be easier to be mainlined now.

**Pull request merged**